test/actor_test.py

```diff
@@ -28,6 +28,8 @@ def ray_start_regular(request):
     # Start the Ray processes.
     ray.init(
         num_cpus=1,
+        redirect_output=True,
```

Comment:
I don't think these are necessary.

Reply:
ah, this was for debugging. I'll remove it.